v4/elements/fig.html

```diff
@@ -62,7 +62,7 @@ <h3 id="fig">&lt;fig&gt;</h3>
          <div class="desc">(figure) – Groups elements representing or containing graphic information such as
             an
             illustration or figure.
-            <div class="chapterLinksBox"><a class="chapterLink" href="/guidelines/v4/content/textencoding.html#figTableFigures">9.2.1 Figures</a>,<a class="chapterLink" href="/guidelines/v4/content/textencoding.html#figTableCaptionsDescriptions">9.2.1.1 Figure Captions and Descriptions</a></div>
+            <div class="chapterLinksBox"><a class="chapterLink" href="/guidelines/v4/content/textencoding.html#figTableFigures">9.2.3 Figures</a>,<a class="chapterLink" href="/guidelines/v4/content/textencoding.html#figTableCaptionsDescriptions">9.2.3.1 Figure Captions and Descriptions</a></div>
```

Comment:
i do not understand your "maybe ..." could you help please.

Reply:
Since this was not included in the original table, I was not sure if you would like to correct this as well ;) We can include it in another PR if you think that goes beyond the scope of this one.